core/src/wallet/tezos/explorers/NodeTezosLikeBlockchainExplorer.cpp

```diff
@@ -250,7 +256,84 @@ namespace ledger {
             );
         }
 
-        Future<std::shared_ptr<BigInt>> NodeTezosLikeBlockchainExplorer::getStorage(const std::string &address) {
+        Future<std::shared_ptr<BigInt>>
+        NodeTezosLikeBlockchainExplorer::getEstimatedGasLimit(
```

Comment:
If I'm not mistaken, the code of `getEstimatedGasLimit` is exactly the same for `ExternalTezosLikeBlockchainExplorer` and `NodeTezosLikeBlockchainExplorer`, so maybe it could belong to `TezosLikeBlockchainExplorer` to avoid duplication?